solarforecastarbiter/io/fetch/__init__.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
```

Comment:
This is a lot of stuff for an `__init__.py` module. Why put it here instead of in its own module?

Reply:
I don't think there is anything wrong with putting a bunch of stuff in ``__init__.py``. Some things could go into a ``utils.py``I guess

Reply:
Not a big deal. I usually assume init files are empty. Seems that there are differences of opinion about this on SO.